src/kvilib/irc/KviAvatar.cpp

```diff
@@ -56,7 +56,9 @@
 		Every IRC user has a client-side property called AVATAR. Let's say that there
 		are two users: A and B.[br]
 		When user A wants to see the B's avatar he simply sends a CTCP AVATAR request
-		to B (the request is sent through a PRIVMSG IRC command).[br]
+		to B (the request is sent through a PRIVMSG IRC command). This is negotiated by
+		hiding an U+0003 '4' U+000F sequence at the start of B's realname, which doesn't
```

Comment:
'4' is not always there. The exact number depends on the gender set. See https://github.com/DarthGandalf/znc/blob/dgmods/modules/kvirc.cpp#L40-L43 where I reimplemented parts of this

Reply:
The "gender" idea, especially with no privacy/mutability options, needs to be deprecated. But okay, providing compatibility with those isn't too bad we guess.

Reply:
Sorry but I'm not merging this. If you don't want to use gender, don't, but the feature is not going anywhere until there is a valid replacement.